src/app/models/climate-model.model.ts

```diff
@@ -2,5 +2,6 @@ export interface ClimateModel {
     name: string;
     description: string;
     base_time: string;
+    enabled?: boolean;
     selected?: boolean;
```

Comment:
also add `datasets: string[];` :)

Reply:
The property is not getting used currently and duplicates the information on the `dataset` object (list of valid models), but for completeness it does make sense to add it.